jumanji/wrappers_test.py

```diff
@@ -228,6 +235,25 @@ def test_jumanji_environment_to_deep_mind_env__unwrapped(
         """Validates unwrapped property of the wrapped environment."""
         assert isinstance(fake_dm_env.unwrapped, Environment)
 
+    def test_jumanji_environment_to_deep_mind_env__key_getter_setter(
+        self, fake_environment: FakeEnvironment)-> None:
+        """validate state getter and setter"""
```

Comment:
NIT: Could we please update docstrings to have them consistent with others? We try to keep them case sensitive.
```suggestion
        """Validates state getter and setter."""
```